scripts/init_kubeconfig.sh

```diff
@@ -2,7 +2,7 @@
 ca=$(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | base64 -w 0)
 token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
 namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
-server=https://10.43.0.1
+server=https://${KUBERNETES_SERVICE_HOST}
```

Comment:
This should work, we could also use `kubernetes.default.svc`, or specify `KUBERNETES_SERVICE_PORT_HTTPS`, but I'd assume it won't matter.

Reply:
Didn't use `kubernetes.default.svc`  to avoid lookup overhead.
While doing some research, found that the [API service port will be 443](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/server.go#L284) always.
```
>kubectl get svc kubernetes -o yaml
...
  ports:
  - name: https
    port: 443 <<------
    protocol: TCP
    targetPort: 6443
...
```
So I hope it is ok to use just `KUBERNETES_SERVICE_HOST`